src/stylesheets/themes/hugo-terminal-orange/selection.scss

```diff
@@ -0,0 +1,15 @@
+@import "./variables";
+
+@mixin selection($element) {
+  #{$element}::-moz-selection { @content; }
```

Comment:
autoprefixer will add the prefixed version of `::selection`. Should be able to remove this line.